classes/core/PKPContainer.inc.php

```diff
@@ -167,8 +171,17 @@ protected function loadConfiguration()
             'retry_after' => 90,
         ];
 
+        // Logging
+        $items['logging']['channels']['mail'] = [
+            'driver' => 'single',
+            'path' => Config::getVar('email', 'debug_path') ??
+                (new PrivateFileManager())->getBasePath() . '/' . 'email.log',
```

Comment:
We'll probably make use of logging for other things eventually. Should we create a `logs` directory in the private files directory? So it would be `<files_dir>/logs/email.log`.